src/main/scala/io/xskipper/metadatastore/parquet/ParquetUtils.scala

```diff
@@ -280,7 +291,7 @@ object ParquetUtils extends Logging {
     // this only works since we need the schema tree (names and types) and not metadata, so
     // we can live with the incorrect md returned in the schema
     val parquetSchema = converter.convert(
-      createDFSchema(indexes, includeExtraColumns = false, "", None))
+      createDFSchema(indexes, None, includeExtraColumns = false, "", None))
```

Comment:
is this fine?

Reply:
yes, updated the comment that explains this.
TL;DR: this call is only to retrieve the column paths for a specific index, and is not influenced by the partition schema in any way - meaning in particular it can be `None`.